### PR TITLE
HKG: Add Car Port for 2021 Kia K5

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added to RELEASES, CARS
- [x] test route added to [test_routes.py](https://github.com/sunnyhaibin/openpilot-1/blob/85dccd4f743ba507865c5a0271812854f0c90a20/selfdrive/test/test_routes.py#L89)
- [x] route with openpilot: `007d5e4ad9f86d13|2021-09-25--11-23-14`

Thanks to community Kia K5 owner 320Pilot.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -547,6 +548,26 @@ class Buttons:
       b"\xf1\x816U2VC051\x00\x00\xf1\x006U2V0_C2\x00\x006U2VC051\x00\x00DBD0T16SS0\xcf\x1e'\xc3",
     ],
   },
+  CAR.KIA_K5_2021: {
+    (Ecu.fwdRadar, 0x7D0, None): [
+      b'\xf1\000DL3_ SCC FHCUP      1.00 1.03 99110-L2000         ',
+    ],
+    (Ecu.eps, 0x7D4, None): [
+      b'\xf1\x8756310-L3110\xf1\000DL3 MDPS C 1.00 1.01 56310-L3110 4DLAC101\xf1\xa01.01',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:556]:
```suggestion
      b'\xf1\x8756310-L3110\xf1\000DL3 MDPS C 1.00 1.01 56310-L3110 4DLAC101',
```

Comment (pd0wm) [reply selfdrive/car/hyundai/values.py:556]: Can you make sure the ECU versions are still accurate after my latest changes to the firmware query?

Comment (sunnyhaibin) [reply selfdrive/car/hyundai/values.py]: Done.

File: selfdrive/test/test_routes.py [hunk 2]
```diff
@@ -86,6 +86,7 @@
   TestRoute("49f3c13141b6bc87|2021-07-28--08-05-13", HYUNDAI.KONA_HEV),
   TestRoute("5dddcbca6eb66c62|2020-07-26--13-24-19", HYUNDAI.KIA_STINGER),
   TestRoute("d624b3d19adce635|2020-08-01--14-59-12", HYUNDAI.VELOSTER),
+  TestRoute("007d5e4ad9f86d13|2021-09-25--11-23-14", HYUNDAI.KIA_K5_2021),
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:89]: this route isn't fully uploaded. connect the device to wifi, then go to https://useradmin.comma.ai/?onebox=007d5e4ad9f86d13%7C2021-09-25--11-23-14 and click "upload all logs" once it pops up

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
This looks good to merge once the test route gets uploaded

File: selfdrive/test/test_routes.py (hunk 2 above)
Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py:89]: hmm, this car fingerprints as a sonata

Comment (320Pilot) [reply selfdrive/test/test_routes.py:89]: Just got home. Yes I changed the fingerprint and FW of the Sonata to the K5 so OP would work. I don't see an "upload all logs" in my drive log .

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py:89]: hmm, looks like those files got deleted. I'm guessing you factory resetted or uninstalled openpilot. we'll need another route, so find a more recent one that has the "upload all logs" button

Comment (320Pilot) [reply selfdrive/test/test_routes.py:89]: ok

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py:89]: If you comment the new route here and keep the device connected to wifi, I'll get the PR updated and route uploaded

Comment (320Pilot) [reply selfdrive/test/test_routes.py:89]: just uploaded  007d5e4ad9f86d13|2021-09-30--15-09-23

Comment (sunnyhaibin) [reply selfdrive/test/test_routes.py]: Test route updated in new commit.

**Comment by adeebshihadeh** [comment]
Thanks for another PR @sunnyhaibin! Next time, can you try to get a test route from the PR branch?:

**Comment by 320Pilot** [comment]
I see K5 Fw in master now, but not the Fingerprint.

**Comment by sunnyhaibin** [comment]
> Thanks for another PR @sunnyhaibin! Next time, can you try to get a test route from the PR branch?

Absolutely 👍

**Pull request merged**